packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -114,7 +114,14 @@ class _LayoutBuilderElement extends RenderObjectElement {
           debugWidgetBuilderValue(widget, built);
         } catch (e, stack) {
           built = ErrorWidget.builder(
-            _debugReportException(ErrorDescription('building $widget'), e, stack, this)
+            _debugReportException(
+              ErrorDescription('building $widget'),
+              e,
+              stack,
+              informationCollector: () sync* {
+                yield DiagnosticsProperty<Object>('offendingContext', this, level: DiagnosticLevel.hidden);
```

Comment:
you made
DebugCreator public so make this
`DiagnosticsProperty<DebugCreator>`
which will let the transformation from debugCreator to a offendingContext more robust.